src/MobileBookingHeader/OneWayTrip.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import * as React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+
+import OneWayTrip_booking from './__generated__/OneWayTrip_booking.graphql';
```

Comment:
`import type` 😱 

Please, check also other files, it's everywhere... 🙂

Reply:
oh wow! Thanks!! :-D Pretty sure I got rid of all of them, no more results for `grep import -r src | grep "__generated__" | grep -v "type"`